src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -2059,6 +2060,7 @@ shared_ptr<runtime::Executable>
         case OP_TYPEID::ScatterAdd:
         case OP_TYPEID::ScatterNDAdd:
         case OP_TYPEID::ShapeOf:
+        case OP_TYPEID::ShuffleChannels:
```

Comment:
Please add "case OP_TYPEID::ShuffleChannels:" line into switch of runtime::intelgpu::IntelGPUBackend::is_supported_impl function (intelgpu_backend.cpp). By #2912 we implemented functionality to decompose unsupported fused ops.

Reply:
No. This line should be it 2 places. In switch of compile function and in switch of is_supported_imp function